src/components/ProposalVoteCard/components/UserVote/UserVote.tsx

```diff
@@ -0,0 +1,46 @@
+import useVotingPowerPercent from 'Modules/Guilds/Hooks/useVotingPowerPercent';
+import { formatUnits } from 'ethers/lib/utils';
+import type { VoteData } from 'Modules/Guilds/Hooks/useVotingResults';
+import { Box } from 'components/primitives/Layout/Box';
+import styled from 'styled-components';
+import { UseProposalVotesOfVoterReturn } from 'Modules/Guilds/Hooks/useProposalVotesOfVoter';
+
+const Container = styled(Box)`

```

Comment:
Should we move this into its own `styles.ts` file?

Reply:
Sure! Will use a `UserVote.styled` file